index.js

```diff
@@ -452,8 +452,20 @@ async function onGithubPullRequest(payload) {
 }
 
 async function onGithubPush(payload) {
+  const { ref, commits } = payload;
   log.info(payload);
-  await Promise.resolve(); // pacify eslint
+  // building only the master branch
+  if (ref !== 'refs/heads/master') {
```

Comment:
I think ideally we want this to be taken from the buildkite pipeline configuration, but this should be totally fine to get us unblocked for now.